mautic/mautic
# Fix date/datetime empty mysql expression for contact field condition

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:

Similar fix for MYSQL8 and date/datetime columns like https://github.com/mautic/mautic/pull/11126
We cannot use empty string condition because MYSQL8 return fatal error


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create campaign with contact field condition on date/datetime field and empty or not empty expression
3.  Try trigger campaign
4. It should failed, because it return MYSQL  error
5. This PR should fixed it
